CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v0.7.0-beta (04/09/2019)
+
+**Common**
+
+* New abstract TaskScheduler introduced to be used for async context
+  (default thread pool implementation provided).
+* Added UserAgent to mark all triggered requests.
+* China Lookup API URL added.
+* New HTTP abstraction layer introduced without singleton usage.
```

Comment:
* Introduced new abstract TaskScheduler to use for async context; provided default thread pool implementation.
* Added UserAgent to mark all triggered requests.
* Added China Lookup API URL.
* Introduced new HTTP abstraction layer without singleton usage.